starknet/docs/gas-service.md

```diff
@@ -7,7 +7,7 @@ This file contains example commands for testing all gas service functions. Repla
 Make sure you have:
 - A funded account on testnet
 - The gas service contract deployed
-- ERC20 tokens for testing gas payments
+- Any ERC20 token for testing gas payments
```

Comment:
I doubt this is going to hold true in the end. I would make it a bit more vague, i.e.: Any _supported_ ERC20 token...

Reply:
Good catch. I'll even add that only STRK is supported for now. 
I know we'll probably not remember to update, in case that changes, but otherwise it's misleading.